.github/workflows/ci_rails.yml

```diff
@@ -0,0 +1,77 @@
+# GitHub Actions Continuous Integration of MO Rails Code
+# Runs the tests and publishes the results to Coveralls
+name: Continuous Integration
+
+on: 
+  push:
+    # branches: master
+    branches: 
+      - "*" # Run on pushes on all branches
+  pull_request:
+    branches: master
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04 # focal, includes MySQL 5.8
+    steps:
+    # check-out repo under $GITHUB_WORKSPACE, so that workflow can access it.
+    # https://github.com/actions/checkout
+    - name: Checkout code
+      uses: actions/checkout@v2  
+
+    # https://github.com/actions/setup-ruby
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true # runs bundle install, caches gems
```

Comment:
I just noticed how slick this is. 
The 1st run on any branch takes 7-8 minutes -- about the same as Travis.
But (due to gem caching) subsequent runs are only about 4 minutes -- about 2x faster than Travis.

Reply:
This seems great! Thanks for doing it Joe.
I'm in the woods today... Can check later